src/components/WebCLI/connection.js

```diff
@@ -52,17 +56,21 @@ class Connection {
   }
 
   _pushToMessageBuffer(message) {
+    const bufferEmpty = !!this._messageBuffer;
     this._messageBuffer = this._messageBuffer + message;
-    setTimeout(() => {
-      /*
+    if (bufferEmpty) {
+      this.timeout = setTimeout(() => {
```

Comment:
Setting the timeout out to a longer delay (e.g. 100) would cause the tests to fail every time.